utils/utils.go

```diff
@@ -115,3 +116,11 @@ func DeploymentNameFromPodName(podName string) (string, error) {
 	}
 	return podName[0:i], nil
 }
+
+func ToJsonOrDie(value interface{}) []byte {
+	if bytes, err := json.Marshal(value); err == nil {
+		return bytes
+	} else {
+		panic(err)
+	}
+}
```

Comment:
If you are adding this here, it can be removed from `testsutils` package and have all test files import it from here.
We don't want to maintain 2 copies

Reply:
Have imported testutils package in managedocs_controller.go and used existing toJsonOrDie function. The altrenative would result in a lots of diff